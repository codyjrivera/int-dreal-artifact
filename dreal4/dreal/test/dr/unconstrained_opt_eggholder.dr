# Eggholder
# https://en.wikipedia.org/wiki/Test_functions_for_optimization#Test_functions_for_single-objective_optimization

var:
    [-512, 512] x;
    [-512, 512] y;

cost:
    -(y + 47) * sin(sqrt(abs(x/2 + y + 47))) - x * sin(sqrt(abs(x - y - 47)));
