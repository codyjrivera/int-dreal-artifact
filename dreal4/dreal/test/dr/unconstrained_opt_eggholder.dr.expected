delta-sat with delta = 2
